usr/lib/anon-shared-helper-scripts/tor_enabled_check

```diff
@@ -23,40 +23,24 @@ check_tor_enabled_do() {
       fi
    fi
 
-   local line file_name file_name_list i
-
    shopt -s globstar
    shopt -s nullglob
+   shopt -s nocasematch
```

Comment:
The changes in this script don't explain why this is needed to be set here. You sure it's required? You sure it should be set here?